pythonforandroid/build.py

```diff
@@ -91,6 +91,9 @@ class Context(object):
     '''A build context. If anything will be built, an instance this class
     will be instantiated and used to hold all the build state.'''
 
+    build_as_debuggable = False
+    # Whether to build with debugging symbols
```

Comment:
super-nitpick: I think we usually put comments the line before or the line in front. Docstrings however goes after

Reply:
oh oops, I actually just misread the later lines thinking they had the comments after and was slightly confused by that :joy: that makes so much more sense